### PR TITLE
fix(EG-726): pages not following tab value provided in query param

### PR DESCRIPTION
**Pull request by willholt-tb**

### CODE REVIEW

File: packages/front-end/src/app/pages/labs/[labId]/[workflowId]/index.vue [hunk 1]
```diff
@@ -70,13 +76,7 @@
...
   });
 
-  onBeforeMount(async () => {
-    await initData();
-
-    let paramTab = $router.currentRoute.value.query?.tab;
-    if (!paramTab) paramTab = 'Run Results'; // fallback for no query param to default to first tab
-    tabIndex = paramTab ? tabItems.findIndex((tab) => tab.label === paramTab) : 0;
-  });
+  onBeforeMount(initData);
```
Comment (simondib) [inline packages/front-end/src/app/pages/labs/[labId]/[workflowId]/index.vue:79]: Should this by async/await?

**Comment by simondib** [comment]
@CodiumAI-Agent /describe
--pr_description.extra_instructions="
For the title, use the format [type]: [summary]
"
--pr_description.generate_ai_title=true
--pr_description.final_update_message=true

File: packages/front-end/src/app/pages/labs/[labId]/[workflowId]/index.vue (hunk 1 above)
Comment (willholt-tb) [reply packages/front-end/src/app/pages/labs/[labId]/[workflowId]/index.vue:79]: it should be awaited - this is a simplification of this:

```
onBeforeMount(async () => {
  await initData();
});
```

which to my typescript knowledge is a redundant anonymous function around initData and `onBeforeMount(initData)` is functionally identical.

Comment (simondib) [reply packages/front-end/src/app/pages/labs/[labId]/[workflowId]/index.vue:79]: Good to know!

**Review by simondib — APPROVED** [review]

**Pull request merged**